src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1016,7 +1006,6 @@ def forward(
         return_loss: Optional[bool] = None,
         output_attentions: Optional[bool] = None,
         output_hidden_states: Optional[bool] = None,
-        return_base_image_embeds: Optional[bool] = None,
```

Comment:
I am fine with all the other removals, but this one is on a public class that has been released more three months, so it needs to undergo at least a couple of minor release in deprecation before being fully removed. I'm fine saying it will be removed in v4.27 of Transformers.

Reply:
Is this ok now? :)